lib/exporter/EnvFileExporter_1_0_0.js

```diff
@@ -453,17 +441,17 @@ class EnvFileExporter_1_0_0 {
       hookObject.type = 'internal';
       let filename;
       if (this.storeCodeAsFile) {
-        filename = 'hooks/' + apiCollectionHook.name + '.json';
+        filename = 'hooks/' + apiCollectionHook.name + '.js';
```

Comment:
String interpolation is much nicer IMO
https://github.com/Kinvey/style-guides/blob/master/backend/es6.md#es6-template-literals

Reply:
Good catch. Agree on both.